meta-avocado-raspberrypi/recipes-avocado/sdk/avocado-sdk-target/stone-raspberrypi4.json

```diff
@@ -1 +1,144 @@
+    "rootdisk": {
+      "out": "avocado-raspberrypi4-rootdisk.zip",
+      "build": "fwup",
+      "build_args": {
+        "template": "rootdisk.conf"
+      },
+      "devpath": "/dev/mmcblk0",
+      "block_size": 512,
+      "images": {
+        "uboot_env": "uboot.env",
+        "boot": {
+          "out": "boot.img",
+          "size": 128,
+          "size_unit": "mebibytes",
+          "build_args": {
+            "type": "fat",
+            "variant": "FAT32"
+          },
+          "files": [
+            "Image",
+            "avocado-image-initramfs-avocado-raspberrypi4.cpio.zst",
+            "bcm2711-rpi-4-b.dtb",
+            "bcm2711-rpi-400.dtb",
+            "bcm2711-rpi-cm4.dtb",
+            "bcm2711-rpi-cm4s.dtb",
+            {
+              "in": "bootfiles/bootcode.bin",
+              "out": "bootcode.bin"
+            },
+            {
+              "in": "bootfiles/cmdline.txt",
+              "out": "cmdline.txt"
+            },
+            {
+              "in": "bootfiles/config.txt",
+              "out": "config.txt"
+            },
+            {
+              "in": "bootfiles/fixup.dat",
+              "out": "fixup.dat"
+            },
+            {
+              "in": "bootfiles/fixup4.dat",
+              "out": "fixup4.dat"
+            },
+            {
+              "in": "bootfiles/fixup4cd.dat",
+              "out": "fixup4cd.dat"
+            },
+            {
+              "in": "bootfiles/fixup4db.dat",
+              "out": "fixup4db.dat"
+            },
+            {
+              "in": "bootfiles/fixup4x.dat",
+              "out": "fixup4x.dat"
+            },
+            {
+              "in": "bootfiles/fixup_cd.dat",
+              "out": "fixup_cd.dat"
+            },
+            {
+              "in": "bootfiles/fixup_db.dat",
+              "out": "fixup_db.dat"
+            },
+            {
+              "in": "bootfiles/fixup_x.dat",
+              "out": "fixup_x.dat"
+            },
+            {
+              "in": "bootfiles/start.elf",
+              "out": "start.elf"
+            },
+            {
+              "in": "bootfiles/start4.elf",
+              "out": "start4.elf"
+            },
+            {
+              "in": "bootfiles/start4cd.elf",
+              "out": "start4cd.elf"
+            },
+            {
+              "in": "bootfiles/start4db.elf",
+              "out": "start4db.elf"
+            },
+            {
+              "in": "bootfiles/start4x.elf",
+              "out": "start4x.elf"
+            },
+            {
+              "in": "bootfiles/start_cd.elf",
+              "out": "start_cd.elf"
+            },
+            {
+              "in": "bootfiles/start_db.elf",
+              "out": "start_db.elf"
+            },
+            {
+              "in": "bootfiles/start_x.elf",
+              "out": "start_x.elf"
+            },
+            {
+              "in": "u-boot.bin",
+              "out": "kernel8.img"
+            }
```

Comment:
Clever. You're renaming `u-boot.bin` to `kernel8.img` to trick the Broadcom bootloader into loading U-Boot instead of the Linux kernel.